src/games/rota/board.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { GameLayout } from 'components/App/Game/GameLayout';
+import { IGameCtx } from 'boardgame.io/core';
+import { IGameArgs } from 'components/App/Game/GameBoardWrapper';
+import { isOnlineGame, isAIGame, isLocalGame } from '../common/gameMode';
+import Typography from '@material-ui/core/Typography';
+import { IG, Phase } from './game';
+import { Field } from './Field';
+
+interface IBoardProps {
+  G: IG;
+  ctx: IGameCtx;
+  moves: any;
+  playerID: string;
+  gameArgs?: IGameArgs;
+}
+
+interface IBoardState {
+  idSelectedPoint: number;
+}
+
+export class Board extends React.Component<IBoardProps, {}> {
+  state: IBoardState = { idSelectedPoint: null };
+
+  _pointClicked = (pointID: number) => {
+    if (this.props.playerID !== this.props.ctx.currentPlayer && !isLocalGame(this.props.gameArgs)) {
+      return;
+    }
+    if (this.props.ctx.phase === Phase.Place) {
+      this.props.moves.placePiece(pointID);
+    } else if (this.state.idSelectedPoint === null) {
```

Comment:
replace with ``} else if (this.state.idSelectedPoint === null && this.props.G.points[pointID].playerID === this.props.ctx.currentPlayer)``

and next else with `} else if (this.state.idSelectedPoint !== null) { ` ?